data/transit/route/train.json

```diff
@@ -99,12 +99,17 @@
       }
     },
     {
-      "displayName": "Auckland Transport",
-      "id": "aucklandtransport-68ffa1",
-      "locationSet": {"include": ["nz"]},
+      "displayName": "AT (Auckland Transport)",
+      "id": "at-8a95ba",
+      "locationSet": {
+        "include": [
+          "nz-auk.geojson",
+          "nz-wko.geojson"
+        ]
+      },
+      "matchNames": ["auckland transport"],
       "tags": {
-        "network": "Auckland Transport",
-        "network:short": "AT",
+        "network": "AT",
```

Comment:
NSI is trying to change `network=AT` to `network=Auckland Transport` again - see https://github.com/osmlab/name-suggestion-index/pull/8182#discussion_r1197786710

I'm not objecting to this change per-se, but it doesn't make sense to only change 1 out of 8 presets, and I think it would be important to prewarn the community that 3,000+ features are about to be retagged (especially since some software uses this tag value)